pkg/process/monitor/process_monitor.go

```diff
@@ -219,11 +257,18 @@ func (pm *ProcessMonitor) mainEventLoop() {
 			time.Sleep(50 * time.Millisecond)
 			if err := pm.initNetlinkProcessEventMonitor(); err != nil {
 				log.Errorf("failed re-initializing process monitor: %s", err)
+				pm.tel.reinitFailed.Add(1)
 				return
 			}
 		case <-logTicker.C:
-			log.Debugf("process monitor stats - total events: %d; exec events: %d; exit events: %d; Channel size: %d; restart counter: %d",
-				pm.eventCount.Swap(0), pm.execCount.Swap(0), pm.exitCount.Swap(0), len(pm.netlinkEventsChannel), pm.restartCounter.Load())
+			log.Debugf("process monitor stats - total events: %d; exec events: %d; exit events: %d; restart counter: %d; max channel size: %d / 2 minutes)",
+				pm.tel.events.Get(),
+				pm.tel.exec.Get(),
+				pm.tel.exit.Get(),
+				pm.tel.restart.Get(),
+				maxChannelSize,
```

Comment:
you can get this for free by using `MetricGroup.Summary()`

Reply:
Thanks